### PR TITLE
FAI-1955 Added Un-Subscribe column

### PR DESCRIPTION
**Pull request by balaji-j**

### CODE REVIEW

File: src/SFA.DAS.FAA.Domain/Entities/SavedSearchEntity.cs [hunk 1]
```diff
@@ -21,6 +22,7 @@ public static implicit operator SavedSearchEntity(SavedSearch source)
                 DateCreated = source.DateCreated,
                 LastRunDate = source.LastRunDate,
                 EmailLastSendDate = source.EmailLastSendDate,
+
```
Comment (killij) [inline src/SFA.DAS.FAA.Domain/Entities/SavedSearchEntity.cs:25]: Did you mean to map the token here?

**Review by killij — COMMENTED** [review]
One minor other point would be the casing, should technically be "UnsubscribeToken".

File: src/SFA.DAS.FAA.Database/Tables/SavedSearch.sql [hunk 2]
```diff
@@ -4,7 +4,8 @@
     [DateCreated]                   datetime            NOT NULL,
     [LastRunDate]                   datetime            NULL,
     [EmailLastSendDate]             datetime            NULL,
-    [SearchParameters]  		    nvarchar(max)       NOT NULL    
+    [SearchParameters]  		    nvarchar(max)       NOT NULL,
+    [UnSubscribeToken]  		    nvarchar(max)       NOT NULL
```
Comment (killij) [inline src/SFA.DAS.FAA.Database/Tables/SavedSearch.sql:8]: Just a thought, would the unsubscribe token field need an index on it, for lookup when someone comes to click on unsubscribe from the email?

File: src/SFA.DAS.FAA.Domain/Entities/SavedSearchEntity.cs (hunk 1 above)
Comment (balaji-j) [reply src/SFA.DAS.FAA.Domain/Entities/SavedSearchEntity.cs:25]: yes, UnusbscibeToken will be added in here

File: src/SFA.DAS.FAA.Database/Tables/SavedSearch.sql (hunk 2 above)
Comment (balaji-j) [reply src/SFA.DAS.FAA.Database/Tables/SavedSearch.sql:8]: No need I guess, we will unpack the UnsubscribeToken using Data Protector to obtain the Unique identifier and fetch the record with that,

No need to pass the token to the APIM -> Inner Api to fetch the entity

Comment (killij) [reply src/SFA.DAS.FAA.Database/Tables/SavedSearch.sql:8]: Ah ok so the token isn't random, gotcha.

**Pull request merged**